src/main/java/com/microsoft/sqlserver/jdbc/AE.java

```diff
@@ -114,6 +131,30 @@ void add(byte[] encryptedKey, int dbId, int keyId, int keyVersion, byte[] mdVers
             assert ((null != cekMdVersion) && (null != mdVersion) && (cekMdVersion.length == mdVersion.length));
         }
     }
+
+    public void add(byte[] encryptedKey, int dbId, int keyId, int keyVersion, byte[] mdVersion, String keyPath,

```

Comment:
I can't figure out where this method is used. It also probably doesn't need to be public.